src/schema/sale/end_sale_mutation.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+
+import { GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { SaleType } from "schema/sale/index"
+
+export default mutationWithClientMutationId({
```

Comment:
We're trying to phase out `default` exports since they're not statically analyzable. Also, maybe rename new files to `.ts` since TypeScript has now been recently implemented?

Reply:
Ah! yeah in that case we should probably refactor all of these at once, in a different PR